ingestion/src/metadata/ingestion/ometa/utils.py

```diff
@@ -71,3 +74,18 @@ def _(arg) -> str:
     Models with __root__
     """
     return str(arg.__root__)
+
+
+def timer(method):
+    def timed(*args, **kw):
+        ts = time.time()
+        result = method(*args, **kw)
+        te = time.time()
+        if "log_time" in kw:
```

Comment:
What is the usage of this? Are we checking the kwargs of the actual function? 🤔

Reply:
Also @harshach , for performance checks usually time.perf_counter() is preferred https://stackoverflow.com/questions/66036844/time-time-or-time-perf-counter-which-is-faster